openwisp_notifications/tests/test_api.py

```diff
@@ -671,6 +798,54 @@ def _unread_notification(notification):
                 '{view}?next={url}'.format(view=reverse('admin:login'), url=url),
             )
 
+    def test_organization_notification_setting_update(self):
+        tester = self._create_user()
+        org = Organization.objects.first()
+
+        with self.subTest('Test for current user'):
+            url = self._get_path(
+                'organization_notification_setting', self.admin.pk, org.pk
+            )
+            response = self.client.post(url, data={'web': True, 'email': True})
+            self.assertEqual(response.status_code, 200)
+            notification_setting = NotificationSetting.objects.filter(
+                user=self.admin, organization_id=org.pk
+            ).first()
+            self.assertTrue(notification_setting.web, True)
+            self.assertTrue(notification_setting.email, True)
```

Comment:
We should assert that all notification settings of the user for this organization was changed. 

```suggestion
        with self.subTest('Test for current user'):
            url = self._get_path(
                'organization_notification_setting', self.admin.pk, org.pk
            )
            NotificationSetting.objects.filter(user=self.admin, organization_id=org.pk).update(email=False, web=False)
            org__setting_count = NotificationSetting.objects.filter(user=self.admin, organization_id=org.pk).count()
            response = self.client.post(url, data={'web': True, 'email': True})
            self.assertEqual(response.status_code, 200)
            self.assertEqual(NotificationSetting.objects.filter(user=self.admin, organization_id=org.pk, email=True, web=True).count(), org_setting_count)
```